design-system/packages/fields/src/Checkbox.tsx

```diff
@@ -27,21 +32,25 @@ type CheckboxControlProps = {
   checked?: boolean;
   /** When true, the checkbox will be disabled. */
   disabled?: boolean;
+  /** The size of the Checkbox */
+  size?: SizeType;
   /** The value of the Checkbox. */
   value?: string;
 } & InputHTMLAttributes<HTMLInputElement>;
```

Comment:
```suggestion
} & Omit<InputHTMLAttributes<HTMLInputElement>, 'size'>;
```

You need to do this because `size?: number` is a prop on input and `string & number` is never.

Reply:
Aaaah thank you 😁